auth-service/build.gradle.kts

```diff
@@ -61,7 +64,8 @@ dependencies {
     // Database
     implementation("org.postgresql:postgresql:42.6.0")
     implementation("com.zaxxer:HikariCP:5.0.1") // Connection pool
-    implementation("org.flywaydb:flyway-core:9.20.1") // Migrations
+    implementation("org.flywaydb:flyway-core:9.20.1")
+    implementation("io.ktor:ktor-server-caching-headers-jvm:2.3.2") // Migrations
```

Comment:
It sometimes does this when building, I'm not sure if it's IntelliJ trying to be helpful or what. If you add the "-jvm" bit to line 53 and delete this one it should be fine.